express-intro/server.ts

```diff
@@ -0,0 +1,13 @@
+import express from 'express';
+
+const app = express();
+
+app.listen(8080, () => {
+  console.log('Express server listening on port 8080');
+});
+
+app.use((req, res, next) => {
+  console.log('Hello, World!');
+  console.log('The date is', new Date());
+  next();
+});
```

Comment:
Hey Eduardo, make sure to put this use middle ware above the listen method call.

listen should be at the bottom of the server file.

Reply:
i just changed it, can you check and let me know if that is better  please and thank you